backend/hubs/models/hub.py

```diff
@@ -382,20 +382,35 @@ class HubThemeColor(models.Model):
         null=True,
         blank=True,
     )
-    lightHover = models.CharField(
-        help_text="lightHover color",
-        verbose_name="lightHover color",
+    extraLight = models.CharField(
+        help_text="extraLight color",
+        verbose_name="extraLight color",
         max_length=200,
         null=True,
         blank=True,
     )
-    extraLight = models.CharField(
-        help_text="extraLight color",
-        verbose_name="extraLight color",
+    contrastText = models.CharField(
+        help_text="contrastText color",
+        verbose_name="contrastText color",
         max_length=200,
         null=True,
         blank=True,
     )
+    background_default = models.CharField(
```

Comment:
We need a separate color for the Navbar. 
The `background_default` field allows us to store this color in the database.
We also plan to fill the background object in the Theme like this:
`background: { default: "", paper: "" }`
The `background_paper` field will define the background color for elements like the project preview.

Reply:
I think we should use the help_text attributes to specify what background_default and background_paper does